content/03.methods.md

```diff
@@ -5,8 +5,72 @@ Talk about dataset - Pubtator
 Talk about preprocessing Pubtator
 Talk about hand annotations for each realtion
 
-## Label Functions
-describe what a label function is and how many we created for each relation
+## Label Functions for Annotating Sentences
+A common challenge in natural language processing is having too few ground truth annotations, even when data are abundant.
+Data programming [@arxiv:1605.07723] circumvents this issue by quickly annotating large datasets by using multiple noisy signals emitted by label functions.
+Label functions are simple pythonic functions emit: a positive label (1), a negative label (-1) or abstain from emitting a label (0).
+We combine these functions using a generative model to output a single annotation, then use a discriminator model to improve upon this model.
+Our label functions fall into these three categories: databases, text patterns and domain heuristics.
+We provide examples for the categories, described below, using the following candidate sentence: \"<span style="color:#02b3e4">PTK6</span> may be a novel therapeutic target for <span style="color:#875442">pancreatic cancer</span>."
```

Comment:
Talk to @vincerubinetti about how to do colors as per https://manubot.github.io/rootstock/v/ed4158c9f2a1b300f82d058be9aefe1f5eb72763/#special